### PR TITLE
Add solvers' language in docs

### PR DESCRIPTION
**Pull request by mtanneau**
Follows a request on the Slack channel to indicate which supported solvers are written in Julia.

* For solvers whose code is on GitHub, I reported the main language as given by GitHub.
* To the best of my knowledge, almost all commercial solvers are written in C. I could not find the information for Knitro, but the [online documention](https://www.artelys.com/docs/knitro/1_introduction/installation.html) mentions C/C++ compilers, so I indicated C/C++.

### CODE REVIEW

**Comment by mlubin** [comment]
Horizontal space is pretty limited for this table. What about putting a long dash in the "Julia package" column if the solver and julia package link to the same page?
It would also be valuable to have a superscipt M after the license for the open-source solvers that require matlab. Whether a solver is written in C or C++ doesn't seem that relevant in this context.

The meaning of both the dash and M should be listed below the table.

File: docs/src/installation.md [hunk 1]
```diff
@@ -28,32 +28,32 @@ is often more complex. We list below the currently available solvers.
...
-| [SeDuMi](http://sedumi.ie.lehigh.edu/)                                         | [SeDuMi.jl](https://github.com/JuliaOpt/SeDuMi.jl)                               | GPL      | LP, SOCP, SDP                      |
-| [Tulip](https://github.com/ds4dm/Tulip.jl)                                     | [Tulip.jl](https://github.com/ds4dm/Tulip.jl)                                    | MPL-2    | LP                                 |
+| Solver                                                                         | Julia Package                                                                    | Language | License  | Supports                           |
+| ------------------------------------------------------------------------------ | -------------------------------------------------------------------------------- | -------- | -------- | ---------------------------------- |
+| [Artelys Knitro](https://www.artelys.com/knitro)                               | [KNITRO.jl](https://github.com/JuliaOpt/KNITRO.jl)                               | C/C++    | Comm.    | LP, MILP, SOCP, MISOCP, NLP, MINLP |
```
Comment (frapac) [inline docs/src/installation.md]: Knitro is entirely written in C.

**Comment by mtanneau** [comment]
> Horizontal space is pretty limited for this table

Not related, but one way to save space could be to write `(MI)LP`, `(MI)SDP`, etc, instead of `LP, MILP, SDP, MISDP`, etc.

> Whether a solver is written in C or C++ doesn't seem that relevant in this context.

I'm quite agnostic here, my reasoning was that if we indicate the language for some, we might as well indicate the language for all.

> What about putting a long dash in the "Julia package" column if the solver and julia package link to the same page?

That sounds reasonable, yes :)

**Comment by matbesancon** [comment]
bump?

**Comment by odow** [comment]
See #2392

**Pull request closed without merging**